### ISSUE TITLE
Extension does not work anymore

### ISSUE THREAD

**Issue by balazser** [rexebin/classio#1]
Something must happened because the extension does not do anything me for a while using VS Code May 2018 (version 1.24) - Typescript. Did you experience same?

config:
 ```
"classio.implementationSymbolColor": "rgba(86,156,214,1)",
  "classio.overrideSymbolColor": "rgba(86,156,214,1)",
  "classio.implementationSymbol": "implement ",
  "classio.overrideSymbol": "override ",
```

**Comment by rexebin** [comment]
Thanks for reporting. It is broken by 1.24 update due to vs code's API changes. I will try to fix it asap.

**Comment by rexebin** [comment]
@balazserdos fixed with 0.3.2. thanks.

**Comment by balazser** [comment]
thank you @rexebin 
like this tool
